LearningHub.Nhs.WebUI/Controllers/AccountController.cs

```diff
@@ -20,6 +20,7 @@
     using LearningHub.Nhs.WebUI.Helpers;
     using LearningHub.Nhs.WebUI.Interfaces;
     using LearningHub.Nhs.WebUI.Models.Account;
+    using LinqToTwitter.Provider;
```

Comment:
Is this needed? Seems like it might have crept in.

Reply:
Removed that namespace, please verify